### DISCUSSION TITLE
Detections Annotations data cannot read.

### DISCUSSION THREAD

**Discussion by Bigbpydai** [rafaelpadilla/review_object_detection_metrics#101] [Q&A]
Hi !

I tried to use Review Object Detection Metrics to evaluate  the perfomance of my object detection model using the Revie Object Detection Metrics, but  this cannot load Detection Annotation data in <class_name, confidence, left,  top,  right,  bottom>(ABSOLUTE)  format and it says "No file was found".

The example of detection data I am using are as follows:

weed 0.54807305 210.06688 536.6663 264.96436 576.56537
weed 0.5408764 2154.1677 2328.391 2209.8494 2368.0244
paddy 0.5188163 1829.8088 574.523 1876.0896 617.053

The extension of the data is ".txt".
If there is a problem with the data, could you please let us know what the problem is?

Best regard.

**Comment by rafaelpadilla** [comment]
Hi,

Please, make the coordinates as integers instead of decimal values.

Let me know if that solves your problem.

**Reply by Bigbpydai** [reply]
Hi,

Thank you for reply and teach the solution! I solved my problem!

Thank you so much !
